components/alert/alert.css

```diff
@@ -0,0 +1,82 @@
+/* || IMPORTS */
+@import url("../../utilities/global.css");
+@import url("../../helper/spacing.css");
+@import url("../../helper/size.css");
+@import url("../../helper/flex.css");
+@import url("../../helper/doc.css");
+
+/* || ROOT CSS */
+:root {
+    --primary-alert-light: #CFE2FF;
+    --secondary-alert-light: #E2E3E5;
+    --success-alert-light: #D1E7DD;
+    --danger-alert-light: #F8D7DA;
+    --warning-alert-light: #FFF3CD;
+    --info-alert-light: #CFF4FC;
+    --primary-alert-dark: #2864be;
+    --secondary-alert-dark: #808080;
+    --success-alert-dark: #17925a;
+    --danger-alert-dark: #ca2b39;
+    --warning-alert-dark: #b89628;
+    --info-alert-dark: #2d96ad;
+}
```

Comment:
Put all the variables in one CSS file, it should be your main CSS file for CL

Reply:
i'm going to separate variables based on component so this is the approach i prefer @JantuDeb  can I go ahead ?

Reply:
I think using all variables in 1 file will reduce redundancies but this works too